ffig/templates/py3.tmpl

```diff
@@ -75,6 +75,7 @@ class {{class.name}}:
     @property
     {% endif %}
     def {{method.name}}(self{{py3_macros.method_parameters(method, leading_comma=True)}}) -> '{{method.return_type|to_hint_type}}':
+  {% if not method.is_noexcept %}
```

Comment:
Inverted logic? Would it be worth swapping the two sections round so it reads as

```
if method.is_noexcept 

...

else

...
```

Reply:
Will do.